core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -535,6 +536,28 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     }
   }
 
+  it should "delete a workspace with linked bond service account" in withTestDataServices { services =>
```

Comment:
TOL: Would it be worth adding a verify to this that it removes all from workspace? Idk if that seems like a critical piece of code, but seems maybe reasonable to test?

Reply:
line 549 would fail if it were not the case.